api/src/main/resources/liquibase.xml

```diff
@@ -225,13 +225,15 @@
     </changeSet>
 
     <changeSet author="slubwama" id="patientflags-1554735240623-13">
+    	<validCheckSum>3:3da6a06d3efc63e8a2d74e3897a486a4</validCheckSum> <!-- old checksum before changing role to patientflags_role -->
+    	<validCheckSum>3:7b9b1c32b280d38e7db301a93991962c</validCheckSum> <!-- new checksum after changing role to patientflags_role -->
         <preConditions onFail="MARK_RAN">
             <not>
-                <indexExists indexName="role"/>
```

Comment:
Why change this index name?

Reply:
In the openmrs schema there already is a relation name role as of the tables. Now when in PostgreSQL we try to create an index named role we get an error : 
`Caused by: org.postgresql.util.PSQLException: ERROR: relation "role" already exists
`
Here is the link to logs: https://pastebin.com/fKaZ5WT1 . This can be because PostgreSQL might be storing index tables in the same schema as that of other tables.

Reply:
Good catch! 👍 

Now what happens to database that already had this index name?

Reply:
The changeset should not cause any issue as I have already added the valid checksums.

Reply:
Doesn't this mean ending up with openmrs instances having different index names?

Reply:
Yes this would lead to different index names. But I don't think it should be an issue since indexes are used internally by db's and never are a part of code. The database system automatically figures out which index to use when we are querying the table.